### PR TITLE
Refactor image_url test from survey and question

### LINKED ISSUE #684: Refactor specs for url validations
question_spec has `validate image_url`
survey_spec has `validate icon_url` and  `validate image_url`

These 3 scenarios are quite similar, please refactor them and reduce the amount of copied code.
Maybe [shared-examples](https://relishapp.com/rspec/rspec-core/docs/example-groups/shared-examples) can be used here

### PR DESCRIPTION
**Pull request by philipeleandro**
fix #684 

Shared test for image_url was created, but I could not use the same shared test for icon_url. Therefore I left icon_url the same way.

@edimossilva review pls

### CODE REVIEW

File: backend/spec/rails_helper.rb [hunk 1]
```diff
@@ -21,7 +21,7 @@
 # directory. Alternatively, in the individual `*_spec.rb` files, manually
 # require only the support files necessary.
 #
-Dir[Rails.root.join('spec/support/**')].sort.each { |f| require f }
```
Comment (philipeleandro) [inline backend/spec/rails_helper.rb:24]: I had to change this line to use shared examples, I don't know if I could do that. So let me know if it's corret.

**Review by edimossilva — CHANGES_REQUESTED** [review]
I believe you can also test icon_url here, for this you will need to pass the field name as parameter

File: backend/spec/rails_helper.rb (hunk 1 above)
Comment (edimossilva) [reply backend/spec/rails_helper.rb:24]: nice!

File: backend/spec/models/question_spec.rb [hunk 2]
```diff
@@ -113,21 +113,9 @@
     end
   end
 
-  describe 'validate image_url' do
-    let!(:question) { create(:question) }
+  describe 'image_url validation' do
+    let(:tested_class) { create(:question) }
```
Comment (edimossilva) [inline backend/spec/models/question_spec.rb:117]: rename this to `object_instance`

**Review by edimossilva — COMMENTED** [review]
almost there

**Review by edimossilva — COMMENTED** [review]
Please squash the commits!

**Review by edimossilva — APPROVED** [review]

**Pull request merged**